packages/react-sample/src/pages/Callback/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useLogto } from '@logto/react';
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const Callback = () => {
+  const { isAuthenticated, isLoading } = useLogto();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (isAuthenticated && !isLoading) {
+      navigate('/', { replace: true });
+    }
+  }, [isAuthenticated, isLoading, navigate]);
+
+  return <p>Redirecting...</p>;
+};
+
```

Comment:
should have `handleSignInCallback()` logic?

Reply:
Not necessary since there is a `useEffect` in the react SDK that automatically calls `handleSignInCallback`.
```typescript
  useEffect(() => {
    if (!isAuthenticated && logtoClient?.isSignInRedirected(window.location.href)) {
      void handleSignInCallback(window.location.href);
    }
  }, [handleSignInCallback, isAuthenticated, logtoClient]);
```

```typescript
  public isSignInRedirected(url: string): boolean {
    const { signInSession } = this;

    if (!signInSession) {
      return false;
    }
    const { redirectUri } = signInSession;

    return url.startsWith(redirectUri);
  }
```

Reply:
Discussed offline and this is OK.